editor/animation_track_editor.h

```diff
@@ -572,6 +575,12 @@ class AnimationTrackEditor : public VBoxContainer {
 
 	void _anim_duplicate_keys(bool transpose);
 
+	void _anim_copy_keys();
+
+	bool _is_track_compatible(int target_track_idx, Variant::Type source_value_type, Animation::TrackType source_track_type);
```

Comment:
```suggestion
	bool _is_track_compatible(int p_target_track_idx, Variant::Type p_source_value_type, Animation::TrackType p_source_track_type);
```
Prefix arguments with `p_`